ros2cli/ros2cli/xmlrpc/__init__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import xmlrpc.client as client
+
+from . import local_server
+from . import marshal
+
+
+__all__ = [
+    'client',
+    'local_server',
+    'marshal'
+]
```

Comment:
Once we import the specific classes used all the content of this file could go away.

Reply:
That's true, but we need code using either server or client to import our custom `xmlrpc` (un)marshalling. This is the way I found to not trigger the linter because of unused imports.

Reply:
`  # noqa: F401`?

Reply:
A comment why the import is there anyway would be helpful I think.

Reply:
I'm somewhat inclined towards shielding client code from having to deal with the quirks of marshalling support registration. PTAL at 7750f7c3798d2c427f7ef434a366b631787951f2, that ensures it is as seamless as possible.

Having said that, if you have a strong opinion against, we can change it to use `noqa` where appropriate.

Reply:
It seems the crucial part is importing `ros2cli/xmlrpc/marshal/rclpy.py` since it has side effects. How about just importing that module directly with a note that its side effect is what is desired.

Reply:
Fair enough, see https://github.com/ros2/ros2cli/pull/493/commits/e923320345bd371c53ca8e01457acfc7ff61bec9.